src/init/element/element.ts

```diff
@@ -62,7 +62,7 @@ export function createElementGenerator(templateName: string):
             if (!nameValidation.isValid) {
               this.log(`\n${nameValidation.message}\nPlease try again.`);
             } else if (nameValidation.message) {
-              this.log(''); // 'empty' log inserts a line break
+              this.log('');  // 'empty' log inserts a line break
```

Comment:
nit: only 1 space needed

Reply:
Hmmm, this is what `npm run format` gave as output

Reply:
Really? I didn't think npm format touched spacing like that. In that case, this is fine :)